### PR TITLE
fix(typeorm): fix query in findUserByServiceId #1108

### PR DESCRIPTION
**Pull request by blessanabraham**
This fixes the find query in `findUserByServiceId`

Fixes: #1108

### CODE REVIEW

File: packages/database-typeorm/src/typeorm.ts [hunk 1]
```diff
@@ -166,11 +166,13 @@ export class AccountsTypeorm implements DatabaseInterface {
   }
 
   public async findUserByServiceId(serviceName: string, serviceId: string): Promise<User | null> {
-    const service = await this.serviceRepository.findOne({
-      name: serviceName,
-      serviceId,
-      cache: this.options.cache,
-    } as any);
+    const service = await this.serviceRepository.findOne(
```
Comment (pradel) [inline packages/database-typeorm/src/typeorm.ts:169]: Should this follow the same pattern than line 111? (with the where condition)
```
    const service = await this.serviceRepository.findOne({
      where: {
        name: 'email.verificationTokens',
        token,
      },
      cache: this.options.cache,
    });
```

Comment (blessanabraham) [reply packages/database-typeorm/src/typeorm.ts:169]: Well, both patterns are used in this file. Personally, I prefer the where clause pattern (like you suggested). I will change it

**Comment by pradel** [comment]
@blessanabraham can you uncomment the tests merged in the other pr? From my understanding, they should work with the fix right?

**Comment by blessanabraham** [comment]
Yes, it's running as I am writing this

**Comment by pradel** [comment]
Thanks for the fix @blessanabraham!

**Pull request merged**